cpu/nrf51822/include/nrfmin.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        drivers_nrf51822_nrfmin NRF Minimal Radio Driver
+ * @ingroup         drivers
+ * @brief           Minimal driver for the NRF51822 radio
+ * @{
+ *
+ * @file
+ * @brief           Interface definition for the nrfmin NRF51822 radio driver
+ *
+ * This driver enables the use of the NRF51822 radio in a 802.15.4 like fashion.
+ * The driver does however not support the 802.15.4 frame format, but uses its
+ * own packet format.
+ *
+ * The driver supports:
+ *   - short address (16-bit)
+ *   - using CPU-ID for default address
+ *   - address broadcast (broadcast address is ff:ff)
+ *   - PAN IDs (0 to 0xffff)
+ *   - setting of channel (0 to 0x3f)
+ *   - setting of TX power (+4dBm to -20dBm)
+ *   - packet type labeling
+ *   - setting device state (RX, SLEEP)
+ *
+ * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef NRFMIN_H_
+#define NRFMIN_H_
+
+#include "net/ng_netdev.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Default PAN ID used after initialization
+ */
+#define NRFMIN_DEFAULT_PAN          (0x0550)
+
+/**
+ * @brief   Default channel set after initialization
+ */
+#define NRFMIN_DEFAULT_CHANNEL      (1U)                /* 2401MHz */

```

Comment:
Please set it to >=4. Bluetooth is starting at channel 4 to take sure there is no interference with frequencies less than 2400MHz.

I prefer channel 49 or 50. This frequency is outside of all WIFI channels and usable world wide. -> http://en.wikipedia.org/wiki/List_of_WLAN_channels